libheif/heif.h

```diff
@@ -966,6 +969,19 @@ struct heif_error heif_context_get_image_handle(struct heif_context* ctx,
                                                 heif_item_id id,
                                                 struct heif_image_handle**);
 
+#if WITH_EXPERIMENTAL_GAIN_MAP
+// Get id of the gain map image of the HEIF file. If no gain map image is available, this
+// method will return heif_suberror_No_item_data.
+LIBHEIF_API
+struct heif_error heif_context_get_gain_map_image_ID(struct heif_context* ctx, heif_item_id* id);
```

Comment:
I assume there can be several gain maps in the file. We should add the color image ID as another parameter.

Reply:
Same for the two functions below.

Reply:
I believe this is intended that we only support one gain map at most, and that gain map is for the primary image. Assume if we have multiple color images and multiple gain maps, and it is likely that users accidentally links one gain map to another gain map, and we can't detect that case.

But I agree that  the standard is still in draft stage and this is not clearly defined. I think we can hold this for a little while and still mark this gain map support as "experimental".